import

```diff
@@ -315,29 +386,142 @@ var importPerson = function(obj) {
     });
     req.write(JSON.stringify(person));
     req.end();
-  }, stats.rows * wait);
+  }, stats.rows * options.wait);
+};
+
+var importPlace = function(obj) {
+  var config = supportedTypes[options.recordType],
+      place = {
+        imported_date: new Date().toISOString(),
+        parent: obj.parent || options.placeUUID
+      };
+  Object.keys(obj).forEach(function(key) {
+    if (key === 'uuid') {
+      place._id = obj[key];
+    } else if (key.match(/^contact\.\w+/) || key.match(/^parent\.\w+/)) {
+      var props = key.split('.');
+      if (!place[props[0]]) {
+        place[props[0]] = {};
+      }
+      if (props[1] === 'uuid') {
+        place[props[0]]._id = obj[key];
+      } else {
+        place[props[0]][props[1]] = obj[key];
+      }
+    } else {
+      place[key] = obj[key];
+    }
+  });
+  place.type = config.doc_type;
+  setTimeout(function() {
```

Comment:
No because I'm doing `rows * options.wait` so second row waits longer than the first because ` rows` is being incremented.  https://github.com/medic/medic-import/pull/7/files/ef8d3d0e14627a0d29428b04354971c8f28bf6ca#diff-93473a7344419b15c4219cc2b6c64c6fR447